### PR TITLE
fix-create-erd: updated readme for creating ERD

### LINKED ISSUE #119: Create ERD using django extensions

### PR DESCRIPTION
**Pull request by najamF18**
Closes #119 
erd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,3 +328,32 @@ Now, for any new html page, we need to do the following:
...
+
+Add the following code 
+```
+GRAPH_MODELS ={
+    'all_applications': True,
+    'graph_models': True,
+     }
+```
+ to settings.py:
+
```
Comment (DeFidelity) [inline README.md:351]: What if this line is written as:
Add the following code to `settings.py` file:
```
GRAPH_MODELS ={
    'all_applications': True,
    'graph_models': True,
     }
```
I think it's more readable this way.

File: README.md [hunk 2]
```diff
@@ -328,3 +328,32 @@ Now, for any new html page, we need to do the following:
...
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+ Go to settings.py and add 
+ ```
+ 'django_extensions',
+ ```
+to INSTALLED_APPS
+
```
Comment (DeFidelity) [inline README.md:342]: Go to `settings.py` file and the following line to `INSTALLED_APPS`:
 ```
 'django_extensions',
 ```
For better readability

File: README.md (hunk 1 above)
Comment (najamF18) [reply README.md:351]: Sure. Will update it.

File: README.md (hunk 2 above)
Comment (najamF18) [reply README.md:342]: Got it

File: README.md [hunk 3]
```diff
@@ -328,3 +328,30 @@ Now, for any new html page, we need to do the following:
...
+ pip install graphviz
+ pip install django-extensions
+ ```
+ Go to ```settings.py``` and add the following line to  ```INSTALLED_APPS```
+ ```
+ 'django_extensions',
+ ```
+
+Add the following code to ```settings.py```
+```
```
Comment (turian) [inline README.md:343]: This is already in settings.py, why are you instructing the dev to do this

File: django_app/settings.py [hunk 4]
```diff
@@ -225,3 +225,9 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
```
Comment (turian) [inline django_app/settings.py:229]: A comment? Cuz no one knows what this stuff is or why it's here if they are new to the project?

File: README.md [hunk 5]
```diff
@@ -328,3 +328,30 @@ Now, for any new html page, we need to do the following:
...
+GRAPH_MODELS ={
+    'all_applications': True,
+    'graph_models': True,
+     }
+```
+
+ Now execute the folowing commands:
```
Comment (turian) [inline README.md:350]: Spelling

File: README.md (hunk 3 above)
Comment (najamF18) [reply README.md:343]: I just added it so the user knows that this specific line is essential to export db structure. But will remove it if necessary.

File: django_app/settings.py (hunk 4 above)
Comment (najamF18) [reply django_app/settings.py:229]: Sure. Will be adding that

File: README.md (hunk 5 above)
Comment (najamF18) [reply README.md:350]: Will fix this

File: README.md [hunk 6]
```diff
@@ -328,3 +328,26 @@ Now, for any new html page, we need to do the following:
...
 ```
+
+## Generate ERD with django extensions
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+
+Add the following code to ```settings.py```
+```
```
Comment (DeFidelity) [inline README.md:339]: Tripple backtick ( ``` ) is for multi-line code insertion and single backtick ( ` ) is for single and inline code reference.
So, I think setttings.py should be in placed inside single backtick.

File: README.md [hunk 7]
```diff
@@ -328,3 +328,26 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## Generate ERD with django extensions
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+
```
Comment (DeFidelity) [inline README.md:337]: On linux, use:

```
sudo apt-get install graphviz

pip install pydotv
```

**Review by DeFidelity — APPROVED** [review]

**Pull request merged**